modules/ocf/manifests/systemd/service.pp

```diff
@@ -3,22 +3,20 @@
   $content = undef,
   $ensure = 'running',
   $enable = true,
+  $suffix = 'service',
 ) {
-  file { "/etc/systemd/system/${title}.service":
+  file { "/etc/systemd/system/${title}.${suffix}":
     source   => $source,
     content  => $content,
     notify   => Exec['systemd-reload'],
-    require  => Package['systemd-sysv'],
```

Comment:
Why is this line being removed?

Reply:
We talked about it in the lab, just putting it here in case future reference is necessary. Exec['systemd-reload'] already handles this dependency.

Reply:
Given that you pointed out this is not immediately obvious, I'm not against keeping it in.